contracts/FastBreakV1.cdc

```diff
@@ -122,13 +122,136 @@ access(all) contract FastBreakV1: NonFungibleToken {
     }
 
     /// Metadata Dictionaries
+    /// Player mappings remain in contract storage (bounded by number of users)
+    /// Old dictionaries kept for existing data (pre-upgrade)
     ///
-    access(self) let fastBreakRunByID:      {String: FastBreakRun}
-    access(self) let fastBreakGameByID:     {String: FastBreakGame}
+    access(self) let fastBreakRunByID:      {String: FastBreakRun}  // Legacy: existing data
+    access(self) let fastBreakGameByID:     {String: FastBreakGame}  // Legacy: existing data
     access(self) let fastBreakPlayerByID:   {UInt64: PlayerData}
     access(self) let playerAccountMapping:  {UInt64: Address}
     access(self) let accountPlayerMapping:  {Address: UInt64}
```

Comment:
These are also unbounded arrays, so you might need more efficient storage for these too. I'd recommend trying to create generic structs that handle these regardless of the type so you don't have to have storage structs for each one. You end up with a lot of boilerplate